_includes/header.html

```diff
@@ -51,7 +51,7 @@
             class="usa-nav__link {% if page.url contains '/create-an-account/' %}usa-current{% endif %}"
             href="{{ '/create-an-account/' | locale_url }}"
           >
-            <span>{{ site.data.[page.lang].settings.nav.create_an_account }}</span>
+            <span class="usa-nav__link__inner">{{ site.data.[page.lang].settings.nav.create_an_account }}</span>
```

Comment:
I kinda wish we didn't have this CSS class, since it's not standard to the design system component, and instead we implement it custom in this project. I could see that being a future source of bugs if someone were to copy the markup as [documented in the design system code samples](https://designsystem.digital.gov/components/header/).

Best case, I think we'd either use the default styling or modify the default styling within the design system itself.

Second best case, I think we could at least update the stylesheet selectors to conform to the documented component code.

For example, changing this selector to `.usa-nav__primary-item > span`:

https://github.com/18F/identity-site/blob/e869f1efc8ac3951d7a14a9866a7dd4c4c4ee077/_sass/components/_nav.scss#L163-L167